services/storage/object/features.yaml

```diff
@@ -1,62 +1,61 @@
-title: CCC Object Storage Features
-category_id: CCC.ObjStor
-type: Storage
-description: This documents the minimual set of features that should be present for a object storage service to be considered for use in financial services ecosystems.
-features:
```

Comment:
This is different from the template. We need title, category_id, type, description here before we specify the common and specific features

Reply:
Thanks! I've just updated the guideline and templates